### PR TITLE
completed es6-modules exercise - Nick Sturz

### PR DESCRIPTION
**Pull request by nsturz**
https://user-images.githubusercontent.com/94485412/187555892-7610d9c5-7f88-407f-b7e5-f58fb8fa94b6.mov

### CODE REVIEW

File: es6-modules/src/components/todo-app.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// const { createElement, noop } = require('../lib');
+// const TodoList = require('./todo-list');
+// const TodoForm = require('./todo-form');
+import { createElement, noop } from '../lib';
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+
+export class TodoApp {
```
Comment (scbowler) [inline es6-modules/src/components/todo-app.js:8]: This should be `export default`

File: es6-modules/src/components/todo-form.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib';
+
+export class TodoForm {
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js:4]: This should be `export default`

File: es6-modules/src/components/todo-form.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib';
```
Comment (scbowler) [inline es6-modules/src/components/todo-form.js:2]: `createElement` should be a "named export" so it should use curly brackets for the import.
```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-list.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
+// const { createElement } = require('../lib');
+import createElement from '../lib';
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js:2]: `createElement` should be a "named export" so it should use curly brackets for the import.
```JAVASCRIPT
import { createElement } from '../lib';
```

File: es6-modules/src/components/todo-list.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+
+export class TodoList {
```
Comment (scbowler) [inline es6-modules/src/components/todo-list.js:4]: This should be `export default`

File: es6-modules/src/lib/create-element.js [hunk 6]
```diff
@@ -0,0 +1,18 @@
+// const toArray = require('./to-array');
+import toArray from '.to-array';
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js:2]: Missing a slash in the file path.
```JAVASCRIPT
import toArray from './to-array';
```

File: es6-modules/src/lib/create-element.js [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+export function createElement(tagName, attributes, children = []) {
```
Comment (scbowler) [inline es6-modules/src/lib/create-element.js:4]: This should be `export default`

File: es6-modules/src/lib/noop.js [hunk 8]
```diff
@@ -0,0 +1,3 @@
+export function noop() {}
```
Comment (scbowler) [inline es6-modules/src/lib/noop.js:1]: Should be `export default`

File: es6-modules/src/lib/to-array.js [hunk 9]
```diff
@@ -0,0 +1,5 @@
+export function toArray(value) {
```
Comment (scbowler) [inline es6-modules/src/lib/to-array.js:1]: Should be `export default`

**Review by scbowler — CHANGES_REQUESTED** [review]
A few updates needed. Make sure you are testing by running the `npm run build` command. Let me know if you have any questions.

**Comment by nsturz** [comment]
addressed all change requests

**Review by scbowler — APPROVED** [review]
Good job implementing the feedback, looks great!

**Pull request merged**